validator/src/main.rs

```diff
@@ -1438,6 +1448,22 @@ pub fn main() {
 
     let contact_debug_interval = value_t_or_exit!(matches, "contact_debug_interval", u64);
 
+    let mut supported_indexes: Vec<IndexType> = vec![];
+    for index in matches
+        .values_of("indexes")
+        .unwrap_or_default()
+        .map(|value| match value {
+            "program-id" => IndexType::ProgramId,
+            "token-owner" => IndexType::TokenOwner,
+            "token-mint" => IndexType::Mint,
+            _ => unreachable!(),
+        })
+    {
+        if !supported_indexes.contains(&index) {
```

Comment:
Just use a HashSet to avoid this block?

Reply:
Done in https://github.com/solana-labs/solana/pull/14212/commits/75eb65a7825c7d115d887a4a4e4a81a83f96f614